src/Roslyn.Suppressions/Roslyn.Suppressions/Roslyn.Suppressions.csproj

```diff
@@ -17,7 +17,32 @@
     <ProjectGuid>{082D5D8E-F914-4139-9AE3-3F48B679E3DA}</ProjectGuid>
     <RootNamespace>SonarLint.VisualStudio.Roslyn.Suppressions</RootNamespace>
     <AssemblyName>SonarLint.VisualStudio.Roslyn.Suppressions</AssemblyName>
+    <RequiresNewtonsoft>true</RequiresNewtonsoft>
   </PropertyGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="System.IO.Abstractions" Version="9.0.4" />
+  </ItemGroup>
+  
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\sonarqube-webclient\SonarQube.Client\SonarQube.Client.csproj" />
```

Comment:
Auto-merge duplication with the code below (there was already a reference to Core project). Also I think it's worth putting the package dependencies next to each other.